service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -83,15 +91,22 @@ void shouldReturnHearingDateDynamicListWhenCmoHasPreviousSelectedValue() {
     @Test
     void shouldReturnCaseManagementOrderWhenProvidedCaseDetails() {
         CaseDetails caseDetails = CaseDetails.builder()
-            .data(ImmutableMap.of("cmoHearingDateList", getDynamicList()))
-            .build();
+            .data(ImmutableMap.of(
+                "cmoHearingDateList", getDynamicList(),
+                "allPartiesCustom", createElementCollection(createUnassignedDirection()),
+                "localAuthorityDirectionsCustom", createElementCollection(createUnassignedDirection()),
+                "cafcassDirectionsCustom", createElementCollection(createUnassignedDirection()),
+                "courtDirectionsCustom", createElementCollection(createUnassignedDirection())
```

Comment:
I think you could clean this up a little with the stream I mentioned in another comment